### ISSUE TITLE
bug: Turbo shutting down: `shutting down dev server`after file edit

### ISSUE THREAD

**Issue by jpainam** [t3-oss/create-t3-turbo#1231]
### Provide environment information

```bash

  System:
    OS: macOS 14.5
    CPU: (10) arm64 Apple M1 Max
    Memory: 1.68 GB / 32.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 22.4.0 - ~/.nvm/versions/node/v22.4.0/bin/node
    Yarn: 1.22.22 - ~/.nvm/versions/node/v22.4.0/bin/yarn
    npm: 10.8.1 - ~/.nvm/versions/node/v22.4.0/bin/npm
    pnpm: 9.12.2 - ~/.nvm/versions/node/v22.4.0/bin/pnpm
```

### Describe the bug

Anytime after a file edit. instead of `hot reload`, the webpage become unavailable. It seems it's a `turbo` issue. just posting it here in case anyone has ever encounter such problem.
In the terminal, i can read `shutting down dev server`

I'm using 

```ts
 "devDependencies": {
...
"@turbo/gen": "2.2.1",
 "turbo": "2.2.3"
...
}
```

### Link to reproduction

https://github.com/t3-oss/create-t3-turbo

### To reproduce

This error just came from nowhere. and it's slowing development. 

### Additional information

N/A

**Comment by jpainam** [comment]
downgrading `pnpm` and `turbo` worked 

```bash
"pnpm@9.10.0",
 "turbo": "2.1.2",
"@turbo/gen": "2.1.2",
```

**Comment by steveninety** [comment]
Been dealing with this for HOURS and then I came across this issue...
But it's still not working for me.
I'm about to go crazy lol.

Edit: downgrading to the versions specified by OP did not do the trick for me.
Then, I downgraded to whatever I had at an older commit, and now the issue has been resolved.
I was about to throw my laptop out the window, glad I didn't have to!

**Comment by noxify** [comment]
This will be fixed in the next turborepo version (2.2.4).
